quesma/quesma/router.go

```diff
@@ -254,9 +253,21 @@ func configureRouter(cfg config.QuesmaConfiguration, sr schema.Registry, lm *cli
 			}
 		}
 		return elasticsearchQueryResult(string(responseBody), httpOk), nil
-	}
+	})
 
-	router.Register(routes.EQLSearch, and(method("GET", "POST"), matchedAgainstPattern(cfg)), eqlHandler)
+	router.Register(routes.IndexPath, and(method("PUT"), matchedAgainstPattern(cfg)), func(ctx context.Context, req *mux.Request) (*mux.Result, error) {
+		body, err := types.ExpectJSON(req.ParsedBody)
+		if err != nil {
+			return nil, err
+		}
+
+		mappings := body["mappings"].(map[string]interface{})
```

Comment:
`PUT` is generally used for Index API and mapping is one of call. I think we should  whether it is mapping request